tc/core/polyhedral/schedule_tree_matcher.h

```diff
@@ -29,11 +29,12 @@ namespace polyhedral {
 // that appear in "domain".
 isl::union_set reductionUpdates(isl::union_set domain, const Scop& scop);
 
-// Does the band member with the given partial schedule correspond
-// to a reduction on all statements with a domain in "domain"?
-bool isReductionMember(
-    isl::union_pw_aff member,
+// Does "prefix" partition "domain" into individual reductions?
+// In particular, do the elements of "domain" access a single tensor
```

Comment:
but the code does not actually check something on access relations...

Reply:
> but the code does not actually check something on access relations...

This follows from the properties of the reductions, as explained
on the "reductions" field of Body.

skimo